contracts/external/cw-fund-distributor/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+pub mod contract;
```

Comment:
```suggestion
#![doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/README.md"))]

pub mod contract;
```

This includes the readme in generated rust documentation.